tools/cellprofiler/cellprofiler.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="cp319" name="Cellprofiler" version="@CP_VERSION@">
+    <description>Run a CellProfiler pipeline</description>
```

Comment:
not sure about this one... is this the one that runs the .cppipe that has been assembled in the tools upstream in the workflow?

```suggestion
    <description>runs a CellProfiler pipeline assembled by all the CellProfiler tools upstream in the workflow</description>
```